pom.xml

```diff
@@ -39,7 +39,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <source.version>1.6</source.version>
+        <source.version>1.8</source.version>
```

Comment:
The `source.version` parameter is being used to set the source and target versions on the compiler. If we change this we will change the supported version of the library to jdk 1.8 +.

You could probably separate the properties (one for compile [1.8] and one for target [1.6]).

Reply:
Although, now that I'm thinking about I don't think it will work. I'll schedule this into the next major version (bumping minimum supported version to Java 8).

Reply:
Sure the change is meant for the next major version.